research/attention_ocr/python/model.py

```diff
@@ -665,13 +669,14 @@ def sname(label):
     # tf.summary.text(sname('text/pr'), pr_text)
     # gt_text = charset_mapper.get_text(data.labels[:max_outputs,:])
     # tf.summary.text(sname('text/gt'), gt_text)
```

Comment:
Can you uncomment this lines and, if necessary, replace tf.summary.text with tf.compat.v1.summary.text?
You can do it as a follow-up PR.

Reply:
Yes, I'll do that in another PR